.travis.yml

```diff
@@ -93,6 +93,24 @@ matrix:
         # Report success since we have overridden default behaviour
         - bash -c "$STATUS" success "Local $NAME testing has passed"
 
+    - env:
+        - TEST="astyle"
+      before_install:
```

Comment:
Unfortunately, Travis doesn't inherit from multiple before_installs.

If you want the status update + pipefail + caching (and probably just future compatibility), you'll need to copy them from the global before_install or move this to the install. Maybe a YAML reference will work?

Reply:
will check, and fix somehow.. I noticed that using default version it complains about options not available but their webpage lists them, so need to get update somehow , will fix